pages/osx/netstat.md

```diff
@@ -0,0 +1,16 @@
+# netstat
+
+> Display network-related information such as open connections, open socket ports, etc.
+> More information: <https://man.freebsd.org/cgi/man.cgi?netstat(1)>.
+
+- Displays the PID and program name listening on the given protocol listen:
```

Comment:
```suggestion
- Display the PID and program name listening on a specific protocol:
```

**Ref**. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood

Reply:
Thank you I read the style guide but wasn't being very attentive:P